vtr_flow/benchmarks/symbiflow/symbiflow_counter.eblif

```diff
@@ -0,0 +1,1132 @@
+# Generated by Yosys 0.9+2406 (git sha1 d8b2d1a2, x86_64-conda_cos6-linux-gnu-gcc 1.23.0.449-a04d0 -fvisibility-inlines-hidden -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -fdebug-prefix-map=/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/symbiflow-yosys_1594197534228/work=/usr/local/src/conda/symbiflow-yosys-0.8_6021_gd8b2d1a2 -fdebug-prefix-map=/data/tmp/archdefs/env/conda/envs/symbiflow_arch_def_base=/usr/local/src/conda-prefix -fPIC -Os -fno-merge-constants)
```

Comment:
What's the story for how to regenerate this file?

Reply:
Because the download script pulls the latest, it is possible (and likely) that this eblif will become out of date automatically.  How will that be resolved?  @HackerFoo has a similiar issue for the PGO VTR issue, consider consulting with each other?

Reply:
We could generate a tarball with eblifs along with each -install tarball, as part of the same build.

Reply:
> We could generate a tarball with eblifs along with each -install tarball, as part of the same build.

I think that makes sense to me.

Reply:
Yeah, I think this is the best way to keep things synchronized.
I'd say that, given we only enable small size counter test, for the time being, we could push this as-is, while we wait for a valid package containing the circuits is generated by arch-defs master. As soon as more circuits are available, we can update the `get_symbiflow_benchmarks` target to get also the circuits automatically and remove the counter test.

Reply:
> I'd say that, given we only enable small size counter test, for the time being, we could push this as-is,

No, I do not believe we should merge this as is.  This will break the VTR CI as soon as the blif and tarball get out of date.

Adding example blifs for testing shouldn't be a heavy lift, and I think doing this correctly such that it won't break the VTR CI is valuable.